dapps/evancore.vue.libs/src/components/permissions-editor/permissions-editor.ts

```diff
@@ -168,6 +184,28 @@ class PermissionsEditor extends mixins(EvanComponent) {
 
     return label;
   }
+
+  /**
+   * Return the sort & filter array for the correct contract id.
+   * @param dataSetId
+   */
+  getSortFilter(contractId: string) {
```

Comment:
Is the contractId needed? In which scenario we are using the same instance for multiple contracts?
In the current implementation, the dataSet name is passed into the function, that will never be a contract id.